### PR TITLE
Add BagelDB to Jamstack

### PR DESCRIPTION
**Pull request by bchalk101**
Adding BagelDB - bageldb.com with description and image to Jamstack

### CODE REVIEW

File: src/site/headless-cms/bageldb.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ - Client libraries, including Flutter and JS
+ - Gridsome Plugins
+ - Real-Time Data
+ - Nested Collections field type
+ 
+ There are many more features that BagelDB offer - see our [docs](https://docs.bageldb.com) for more: 
+ 
+## Get Started
+
+Signup to BagelDB [here](app.bageldb.com/signup)
```
Comment (huguestennier) [inline src/site/headless-cms/bageldb.md:29]: This needs to be an absolute link

**Review by huguestennier — CHANGES_REQUESTED** [review]
Looking good! Can you fix the Signup link? and we'll be able to add it in.

File: src/site/headless-cms/bageldb.md (hunk 1 above)
Comment (bchalk101) [reply src/site/headless-cms/bageldb.md]: Update to an absolute link

**Comment by bchalk101** [comment]
Hi peeps, wondering if I've missed anything that still requires changes?

**Review by zachleat — APPROVED** [review]

**Comment by zachleat** [comment]
Merged, Thank you!

**Pull request merged**